diff --git a/node_modules/@react-pdf/pdfkit/lib/pdfkit.browser.js b/node_modules/@react-pdf/pdfkit/lib/pdfkit.browser.js
index 6751116..b517bf9 100644
--- a/node_modules/@react-pdf/pdfkit/lib/pdfkit.browser.js
+++ b/node_modules/@react-pdf/pdfkit/lib/pdfkit.browser.js
@@ -46887,6 +46887,13 @@ var ImagesMixin = {
 
 var AnnotationsMixin = {
   annotate(x, y, w, h, options) {
+    const structureElement = options.structureElement;
+    delete options.structureElement;
+    if (structureElement) {
+      const structParent = this.createStructParentTreeNextKey();
+      options.StructParent = structParent;
+      this.getStructParentTree().add(structParent, structureElement.dictionary);
+    }
     options.Type = 'Annot';
     options.Rect = this._convertRect(x, y, w, h);
     options.Border = [0, 0, 0];
@@ -46911,6 +46918,9 @@ var AnnotationsMixin = {
     const ref = this.ref(options);
     this.page.annotations.push(ref);
     ref.end();
+    if (structureElement) {
+      structureElement.add(new PDFStructureObject(this.page.dictionary, ref));
+    }
     return this;
   },
   note(x, y, w, h, contents, options) {
@@ -47180,6 +47190,13 @@ class PDFStructureContent {
   }
 }
 
+class PDFStructureObject {
+  constructor(pageRef, objectRef) {
+    this.pageRef = pageRef;
+    this.objectRef = objectRef;
+  }
+}
+
 /*
 PDFStructureElement - represents an element in the PDF logical structure tree
 By Ben Schmidt
@@ -47294,7 +47311,7 @@ class PDFStructureElement {
     this._flush();
   }
   _isValidChild(child) {
-    return child instanceof PDFStructureElement || child instanceof PDFStructureContent || typeof child === 'function';
+    return child instanceof PDFStructureElement || child instanceof PDFStructureContent || child instanceof PDFStructureObject || typeof child === 'function';
   }
   _contentForClosure(closure) {
     const content = this.document.markStructureContent(this.dictionary.data.S);
@@ -47314,6 +47331,9 @@ class PDFStructureElement {
       if (child instanceof PDFStructureElement) {
         return child._isFlushable();
       }
+      if (child instanceof PDFStructureObject) {
+        return true;
+      }
       return true;
     });
   }
@@ -47356,6 +47376,13 @@ class PDFStructureElement {
         }
       });
     }
+    if (child instanceof PDFStructureObject) {
+      this.dictionary.data.K.push({
+        Type: 'OBJR',
+        Pg: child.pageRef,
+        Obj: child.objectRef,
+      });
+    }
   }
 }
 
@@ -49629,13 +49656,14 @@ var MetadataMixin = {
     this.metadata.append(xml, newline);
   },
   _addInfo() {
+    const description = this.info.Description || this.info.Subject;
     this.appendXML(`
         <rdf:Description rdf:about="" xmlns:xmp="http://ns.adobe.com/xap/1.0/">
             <xmp:CreateDate>${this.info.CreationDate.toISOString().split('.')[0] + 'Z'}</xmp:CreateDate>
             <xmp:CreatorTool>${this.info.Creator}</xmp:CreatorTool>
         </rdf:Description>
         `);
-    if (this.info.Title || this.info.Author || this.info.Subject) {
+    if (this.info.Title || this.info.Author || description) {
       this.appendXML(`
             <rdf:Description rdf:about="" xmlns:dc="http://purl.org/dc/elements/1.1/">
             `);
@@ -49657,14 +49685,14 @@ var MetadataMixin = {
                 </dc:creator>
                 `);
       }
-      if (this.info.Subject) {
+      if (description) {
         this.appendXML(`
-                <dc:description>
-                    <rdf:Alt>
-                        <rdf:li xml:lang="x-default">${this.info.Subject}</rdf:li>
-                    </rdf:Alt>
-                </dc:description>
-                `);
+            <dc:description>
+                <rdf:Alt>
+                    <rdf:li xml:lang="x-default">${description}</rdf:li>
+                </rdf:Alt>
+            </dc:description>
+            `);
       }
       this.appendXML(`
             </rdf:Description>
diff --git a/node_modules/@react-pdf/pdfkit/lib/pdfkit.js b/node_modules/@react-pdf/pdfkit/lib/pdfkit.js
index 3619d74..1fb21f7 100644
--- a/node_modules/@react-pdf/pdfkit/lib/pdfkit.js
+++ b/node_modules/@react-pdf/pdfkit/lib/pdfkit.js
@@ -39172,6 +39172,13 @@ var ImagesMixin = {
 
 var AnnotationsMixin = {
   annotate(x, y, w, h, options) {
+    const structureElement = options.structureElement;
+    delete options.structureElement;
+    if (structureElement) {
+      const structParent = this.createStructParentTreeNextKey();
+      options.StructParent = structParent;
+      this.getStructParentTree().add(structParent, structureElement.dictionary);
+    }
     options.Type = 'Annot';
     options.Rect = this._convertRect(x, y, w, h);
     options.Border = [0, 0, 0];
@@ -39196,6 +39203,9 @@ var AnnotationsMixin = {
     const ref = this.ref(options);
     this.page.annotations.push(ref);
     ref.end();
+    if (structureElement) {
+      structureElement.add(new PDFStructureObject(this.page.dictionary, ref));
+    }
     return this;
   },
   note(x, y, w, h, contents, options) {
@@ -39465,6 +39475,13 @@ class PDFStructureContent {
   }
 }
 
+class PDFStructureObject {
+  constructor(pageRef, objectRef) {
+    this.pageRef = pageRef;
+    this.objectRef = objectRef;
+  }
+}
+
 /*
 PDFStructureElement - represents an element in the PDF logical structure tree
 By Ben Schmidt
@@ -39579,7 +39596,7 @@ class PDFStructureElement {
     this._flush();
   }
   _isValidChild(child) {
-    return child instanceof PDFStructureElement || child instanceof PDFStructureContent || typeof child === 'function';
+    return child instanceof PDFStructureElement || child instanceof PDFStructureContent || child instanceof PDFStructureObject || typeof child === 'function';
   }
   _contentForClosure(closure) {
     const content = this.document.markStructureContent(this.dictionary.data.S);
@@ -39599,6 +39616,9 @@ class PDFStructureElement {
       if (child instanceof PDFStructureElement) {
         return child._isFlushable();
       }
+      if (child instanceof PDFStructureObject) {
+        return true;
+      }
       return true;
     });
   }
@@ -39641,6 +39661,13 @@ class PDFStructureElement {
         }
       });
     }
+    if (child instanceof PDFStructureObject) {
+      this.dictionary.data.K.push({
+        Type: 'OBJR',
+        Pg: child.pageRef,
+        Obj: child.objectRef,
+      });
+    }
   }
 }
 
@@ -41912,13 +41939,14 @@ var MetadataMixin = {
     this.metadata.append(xml, newline);
   },
   _addInfo() {
+    const description = this.info.Description || this.info.Subject;
     this.appendXML(`
         <rdf:Description rdf:about="" xmlns:xmp="http://ns.adobe.com/xap/1.0/">
             <xmp:CreateDate>${this.info.CreationDate.toISOString().split('.')[0] + 'Z'}</xmp:CreateDate>
             <xmp:CreatorTool>${this.info.Creator}</xmp:CreatorTool>
         </rdf:Description>
         `);
-    if (this.info.Title || this.info.Author || this.info.Subject) {
+    if (this.info.Title || this.info.Author || description) {
       this.appendXML(`
             <rdf:Description rdf:about="" xmlns:dc="http://purl.org/dc/elements/1.1/">
             `);
@@ -41940,11 +41968,11 @@ var MetadataMixin = {
                 </dc:creator>
                 `);
       }
-      if (this.info.Subject) {
+      if (description) {
         this.appendXML(`
                 <dc:description>
                     <rdf:Alt>
-                        <rdf:li xml:lang="x-default">${this.info.Subject}</rdf:li>
+                        <rdf:li xml:lang="x-default">${description}</rdf:li>
                     </rdf:Alt>
                 </dc:description>
                 `);
